### PR TITLE
add mesos-demo example

### PR DESCRIPTION
**Pull request by amreshakim**
Signed-off-by: Amre Shakimov <amre@covalent.io>

### CODE REVIEW

**Review by tgraf — APPROVED** [review]

**Comment by ianvernon** [comment]
This looks good. @tgraf  should we try to integrate this in our CI?

**Review by ianvernon — APPROVED** [review]

**Comment by aanm** [comment]
>This looks good. @tgraf should we try to integrate this in our CI?

Yes we definitely should

File: examples/mesos/README.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    sudo service mesos-slave restart
+
+### tail\_client.sh does not print anything
+
+Make sure you regenerated the client.json if you restarted a web-server task. Inspect client.json
+and see if it uses the IP of the web-server.
+
+Run
+
+    cilium endpoint list
```
Comment (aanm) [inline examples/mesos/README.md:90]: Can you provide an example on what the user might expect to see?

Comment (amreshakim) [reply examples/mesos/README.md:90]: @danwent wants to rewrite this README completely so I will just leave it as it is.

**Pull request merged**